src/Conf.ml

```diff
@@ -711,6 +686,20 @@ let no_version_check =
   let default = false in
   mk ~default Arg.(value & flag & info ["no-version-check"] ~doc ~docs)
 
+let config =
+  let doc =
+    "Aggregate options. Options are specified as a list of comma-separated \
```

Comment:
I would have written "a comma-separated list", but don't know a grammatical justification for it